File: tools/source/sdks/cpp-sdk/examples/health.cpp
#include <iostream>
#include "../src/game_agent_engine_client.hpp"

int main() {
    GameAgentEngineClient client("http://127.0.0.1:8080", "dev-key");
    std::cout << client.healthPath() << std::endl;
    return 0;
}

File: tools/source/sdks/js-sdk/examples/health.js
const { GameAgentEngineClient } = require('../src/client');

async function main() {
  const client = new GameAgentEngineClient(process.env.GAE_SERVER || 'http://127.0.0.1:8080', process.env.GAE_KEY || 'dev-key');
  console.log(await client.health());
}

main().catch((err) => {
  console.error(err);
  process.exit(1);
});
